### PR TITLE
Add url flag

### PR DESCRIPTION
**Pull request by jpreese**
Add the `url` flag to the `doc` command which allows users to set the Url where the policies are hosted at, rather than always assuming a relative directory.

### CODE REVIEW

File: internal/commands/document.go [hunk 1]
```diff
@@ -36,13 +36,20 @@ func newDocCommand() *cobra.Command {
 	konstraint doc
 
 Save the documentation to a specific directory
-	konstraint doc --output docs/policies.md`,
+	konstraint doc --output docs/policies.md
+	
+Set the Url where the policies are hosted at
```
Comment (jalseth) [inline internal/commands/document.go:41]: nit: URL should be all capital letters here.

File: internal/commands/document.go [hunk 2]
```diff
@@ -36,13 +36,20 @@ func newDocCommand() *cobra.Command {
...
+	konstraint doc --url github.com/plexsystems/konstraint`,
```
Comment (jalseth) [inline internal/commands/document.go:42]: nit: URLs should include the protocol (`https://`)

File: internal/commands/document.go [hunk 3]
```diff
@@ -53,6 +60,7 @@ Save the documentation to a specific directory
 	}
 
 	cmd.Flags().StringP("output", "o", "policies.md", "Output location (including filename) for the policy documentation")
+	cmd.Flags().String("url", "", "The Url where the policy files are hosted at (e.g. github.com/policies")
```
Comment (jalseth) [inline internal/commands/document.go:63]: nit: See previous nits.

**Comment by jpreese** [comment]
Whooops, good catches. All set!

**Pull request merged**